vhdbuilder/packer/vhd-scanning.sh

```diff
@@ -71,6 +73,8 @@ if [[ "${OS_TYPE}" == "Linux" && "${ENABLE_TRUSTED_LAUNCH}" == "True" ]]; then
     VM_OPTIONS+=" --security-type TrustedLaunch --enable-secure-boot true --enable-vtpm true"
 fi
 
+SCANNING_ID=$(az identity show --ids "${SCANNING_MSI_RESOURCE_ID}" --query id --output tsv)
```

Comment:
why are we doing this? SCANNING_ID will have the same value as SCANNING_MSI_RESOURCE_ID after this call

further, the call to create the VM should fail with an error saying the identity doesn't exist, if your aim here was to check for that

Reply:
Oh you're right - I was initially looking up the identity by RG and name and getting the principal ID, to assign that. But I I'm just getting the ID directly passed in, so I don't need to do any lookup. I'll update it.